elements/core/gnome-shell.bst

```diff
@@ -3,16 +3,29 @@
 kind: meson
 
 sources:
-- kind: tar
-  url: gnome_downloads:gnome-shell/48/gnome-shell-48.3.tar.xz
-  ref: fb0203fc748593f14e51732618e1f042525fd719764a0fdb0ee3f6fe413a9b2b
+- kind: git_repo
+  url: github:Kekun/eos-gnome-shell.git
+  ref: 084d581366630c31914fb6e8d3a7c09b6c378111
+- kind: git_module
+  url: gnome:gi-docgen.git
+  path: subprojects/gi-docgen
+  ref: 58433f82d1187f11c25311c6c68c44739d9b3b00
+- kind: git_module
+  url: gnome:libgnome-volume-control.git
+  path: subprojects/gvc
+  ref: 5f9768a2eac29c1ed56f1fbb449a77a3523683b6
+- kind: git_module
+  url: github:ptomato/jasmine-gjs.git
+  path: subprojects/jasmine-gjs
+  ref: 856465dddbd92e82e574891e1ebc79e17d7b708a
```

Comment:
Surprised that the build dependency `gnome-build-meta.bst:core-deps/jasmine-gjs.bst` does not help here